astropy/time/tests/test_corrs.py

```diff
@@ -60,6 +61,57 @@ def test_arrays(self):
         assert bval_arr[0] - bval1 < 1.0 * u.us
         assert bval_arr[1] - bval2 < 1.0 * u.us
 
+    def test_array_skycoords(self):
```

Comment:
I am fairly sure we already test broadcasting of `Time` with `SkyCoord`, i.e., that this is superfluous. And it *is* a maintenance burden to have these extra tests (indeed, we're well overdue in going over our test suite and sanitizing it...).